OCP/app/logger.rb

```diff
@@ -1,5 +1,5 @@
 class Logger
-  def initialize(format, delivery)
+  def initialize(format: format, delivery: delivery)
```

Comment:
Nat não precisa passar assim no initialize
```suggestion
  def initialize(format:, delivery:)
```

Reply:
Corrigido! :D